web/html/topic_row.html

```diff
@@ -0,0 +1,22 @@
+{{ range . }}
+<tr class="border-b hover:bg-gray-100">
+    <td class="py-4 px-6">{{.Code}}</td>
+    <td class="px-6 text-blue-500 hover:underline"><a href="#">{{.Name}}</a></td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td class="space-x-4">
+        <button class="text-blue-500 hover:text-red-500 hover:scale-110" hx-get="/edit-topic?id={{.ID}}"
```

Comment:
This class is used in multiple places. Let's create a common class and reuse it everywhere for consistency.